orttraining/orttraining/python/training/utils/hooks/merge_activation_summary.py

```diff
@@ -102,10 +124,9 @@ def parse_arguments():
     parser.add_argument(
         "--overwrite",
         required=False,
-        action="store_true",
+        action="store_false",
```

Comment:
I think `store_true` is the one we want. When using `store_true`, if `--overwrite` is unspecified, it defaults to `False`; otherwise, it is `True`.

Reply:
Thanks man. My bad.